### PR TITLE
Template update

### PR DESCRIPTION
**Pull request by duarte-castano**

### CODE REVIEW

File: examples/jenkins/Windows.Jenkinsfile [hunk 1]
```diff
@@ -37,10 +37,8 @@ pipeline {
     stage('Install Python Dependencies') {
       steps {
         echo "Create ${env.ArtifactsFolder} Folder"
+        // Create folder for storing artifacts
         powershell "mkdir ${env.ArtifactsFolder}"
-        // Only the virtual environment needs to be installed at the system level
-        echo "Install Python Virtual environments"
-        powershell 'pip install -q -I virtualenv --user'
```
[the remark this answers is not in the document]
Comment (duarte-castano) [reply examples/jenkins/Windows.Jenkinsfile]: Reverted

File: examples/jenkins/Linux.Jenkinsfile [hunk 2]
```diff
@@ -36,39 +36,36 @@ pipeline {
...
+        sh 'pip3 install -q -I virtualenv'
+        // Install the rest of the dependencies
+        withPythonEnv('python3') {
           echo "Install Python requirements"
           sh 'pip3 install -U outsystems-pipeline'
         }
       }
     }
-    stage('Get and Deploy Latest Tags') {
+    stage('Get Latest Tags') {
```
Comment (rrmendes) [inline examples/jenkins/Linux.Jenkinsfile]: Rename the stage to "Get and Deploy Latest Tags" to keep consistency with Windows version.

File: examples/jenkins/Linux.Jenkinsfile [hunk 3]
```diff
@@ -36,39 +36,36 @@ pipeline {
...
         // Only the virtual environment needs to be installed at the system level
         echo "Install Python Virtual environments"
-        sh 'pip3 install -q -I virtualenv --user'
-        // Install the rest of the dependencies at the environment level and not the system level
-        withPythonEnv('python') {
+        sh 'pip3 install -q -I virtualenv'
+        // Install the rest of the dependencies
+        withPythonEnv('python3') {
           echo "Install Python requirements"
           sh 'pip3 install -U outsystems-pipeline'
```
Comment (rrmendes) [inline examples/jenkins/Linux.Jenkinsfile]: 'OSPackageVersion' variable is not being used in this template.

File: examples/jenkins/Linux.Jenkinsfile [hunk 4]
```diff
@@ -11,8 +11,8 @@ pipeline {
     // Artifacts Folder
     ArtifactsFolder = "Artifacts"
     // LifeTime Specific Variables
-    LifeTimeHostname = 'lifetime.acmecorp.com'
     LifeTimeAPIVersion = 2
+    LifeTimeEnvironmentURL = 'lifetime.acmecorp.com'
```
Comment (rrmendes) [inline examples/jenkins/Linux.Jenkinsfile]: Rename variable to "LifeTimeHostname" to keep consistency with Windows version.

File: examples/jenkins/Linux.Jenkinsfile [hunk 5]
```diff
@@ -36,39 +36,36 @@ pipeline {
...
       steps {
-        echo "Create ${env.ArtifactsFolder} Folder"
+        echo "Create Artifacts Folder"
         sh "mkdir ${env.ArtifactsFolder}"
         // Only the virtual environment needs to be installed at the system level
         echo "Install Python Virtual environments"
-        sh 'pip3 install -q -I virtualenv --user'
-        // Install the rest of the dependencies at the environment level and not the system level
-        withPythonEnv('python') {
+        sh 'pip3 install -q -I virtualenv'
```
Comment (rrmendes) [inline examples/jenkins/Linux.Jenkinsfile]: Is the `--user` parameter needed here?

File: examples/jenkins/Linux.Jenkinsfile [hunk 6]
```diff
@@ -77,21 +74,22 @@ pipeline {
...
           sh(script: "python3 -m outsystems.pipeline.evaluate_test_results --artifacts \"${env.ArtifactsFolder}\"", returnStatus: true)
         }
       }
       post {
         always {
-          withPythonEnv('python') {
+          withPythonEnv('python3') {
             echo "Publishing JUnit test results..."
-            junit(testResults: "${env.ArtifactsFolder}/junit-result.xml", allowEmptyResults: true)
+            junit(testResults: "${env.ArtifactsFolder}\\junit-result.xml", allowEmptyResults: true)
```
Comment (rrmendes) [inline examples/jenkins/Linux.Jenkinsfile]: Being a Linux template, maybe we should use a '/' instead here?
```suggestion
            junit(testResults: "${env.ArtifactsFolder}/junit-result.xml", allowEmptyResults: true)
```

File: examples/jenkins/Linux.Jenkinsfile (hunk 3 above)
Comment (duarte-castano) [reply examples/jenkins/Linux.Jenkinsfile]: Template updated to lock package version

File: examples/jenkins/Linux.Jenkinsfile (hunk 2 above)
Comment (duarte-castano) [reply examples/jenkins/Linux.Jenkinsfile]: Stage renamed

File: examples/jenkins/Linux.Jenkinsfile (hunk 4 above)
Comment (duarte-castano) [reply examples/jenkins/Linux.Jenkinsfile]: Variable renamed

File: examples/jenkins/Linux.Jenkinsfile (hunk 6 above)
Comment (duarte-castano) [reply examples/jenkins/Linux.Jenkinsfile]: You're right! template updated

File: examples/jenkins/Linux.Jenkinsfile (hunk 5 above)
Comment (duarte-castano) [reply examples/jenkins/Linux.Jenkinsfile]: Yes! template updated

**Pull request merged**